config.js

```diff
@@ -88,6 +88,21 @@ module.exports = {
     // this will also cause sampling decisions made by other distributed components to be
     // ignored. This is useful for aggregating traces generated by different cloud platform
     // projects.
-    ignoreContextHeader: false
+    ignoreContextHeader: false,
+
+    // A path to a key file relative to the current working directory. If this
+    // field is set, the contents of the pointed file will be used for
+    // authentication instead of your application default credentials.
+    keyFile: null,
+
+    // An alias for keyFile. If both keyFile and keyFilename are set, the value
+    // of keyFilename will be ignored.
+    keyFilename: null,
```

Comment:
Not a requirement - `keyFilename` is part of google-cloud's config options so we should use that. I don't remember where I saw `keyFile` from, I thought it was part of a public interface in another library. I can remove it.